.build/docker/_docker_run.sh

```diff
@@ -26,10 +26,14 @@
 #
 ################################
 
+[ $DEBUG ] && set -x
+
 # variables, with defaults
 [ "x${cassandra_dir}" != "x" ] || cassandra_dir="$(readlink -f $(dirname "$0")/../..)"
 [ "x${build_dir}" != "x" ] || build_dir="${cassandra_dir}/build"
+[ "x${m2_dir}" != "x" ] || m2_dir="${HOME}/.m2/repository"
```

Comment:
Does it work on Jenkins? HOME directory wasn't writable for agent, that's why I moved it

Reply:
yes, because jenkins defines it here: https://github.com/apache/cassandra/pull/2852/files#diff-917491029f94ac1ce6c114a542d2a8738ce1f2d1e866eaffe7189d2139250200R322
and here: https://github.com/apache/cassandra/pull/2852/files#diff-917491029f94ac1ce6c114a542d2a8738ce1f2d1e866eaffe7189d2139250200R362

Reply:
Thank you